apps/namadillo/src/atoms/integrations/functions.ts

```diff
@@ -43,21 +44,20 @@ import internalDevnetChain from "namada-chain-registry/_testnets/namadainternald
 import namadaAssets from "namada-chain-registry/namada/assetlist.json";
 import namadaChain from "namada-chain-registry/namada/chain.json";
 
-import campfireOsmosisTestnetIbc from "namada-chain-registry/_testnets/_IBC/namadacampfire-osmosistestnet.json";
-import housefireOldCosmosTestnetIbc from "namada-chain-registry/_testnets/_IBC/namadahousefireold-cosmoshubtestnet.json";
-import housefireOldOsmosisTestnetIbc from "namada-chain-registry/_testnets/_IBC/namadahousefireold-osmosistestnet.json";
 import internalDevnetCosmosTestnetIbc from "namada-chain-registry/_testnets/_IBC/namadainternaldevnet-cosmoshubtestnet.json";
 
 // TODO: this causes a big increase on bundle size. See #1224.
 import cosmosRegistry from "chain-registry";
+import { searchNamadaTestnetByChainId } from "lib/chain";
 
-cosmosRegistry.chains.push(
+export const namadaTestnetChainList = [
   internalDevnetChain,
   campfireChain,
   housefireChain,
   housefireOldChain,
-  namadaChain
-);
+] as Chain[];
+
+cosmosRegistry.chains.push(...namadaTestnetChainList, namadaChain);
```

Comment:
I vaguely remember having a `find()` in the code that captures the first item on array, and it needs to be the mainnet, maybe an asset related?

Do you think we could invert parameters to be `.push(namadaChain, ...namadaTestnetChainList)`?

Reply:
Hmmm, I can do it, but this `find()` is not right :/ if we find this case, we should refactor